### PR TITLE
Docs/how to contribute a new expectation -> self_check

### PR DESCRIPTION
**Pull request by eugmandel**

### CODE REVIEW

File: examples/expectations/column_aggregate_expectation_template.py
Comment (abegong) [inline examples/expectations/column_aggregate_expectation_template.py:1]: What state is this file currently in?

If it doesn't work, we shouldn't merge it in this PR.

File: examples/expectations/column_map_expectation_template.py
Comment (abegong) [inline examples/expectations/column_map_expectation_template.py]: Let's drop the TODO list, now that that's captured elsewhere.

File: docs/guides/how_to_guides/creating_and_editing_expectations/how_to_contribute_a_new_expectation_to_the_shared_library.rst
Comment (abegong) [inline docs/guides/how_to_guides/creating_and_editing_expectations/how_to_contribute_a_new_expectation_to_the_shared_library.rst:1]: I'm not reviewing this doc closely in this PR--I assume that we'll pick up any issues in UAT and iteration around the hackathons.

**Review by abegong — CHANGES_REQUESTED** [review]

**Comment by abegong** [comment]
Also, why are tests not passing?

File: examples/expectations/column_aggregate_expectation_template.py
Comment (eugmandel) [reply examples/expectations/column_aggregate_expectation_template.py:1]: This template works - no reason not to include it. Obviously, it can be improved, which we will do in the next few days.

Comment (abegong) [reply examples/expectations/column_aggregate_expectation_template.py:1]: Awesome, I just wasn't sure about its state.

**Review by abegong — APPROVED** [review]
lgtm, as soon as tests pass

**Comment by abegong** [comment]
Before merging this, please make sure that it's not going to create duplicate/conflicting files.

It looks like some of the docs files already exist in #2222 . I haven't done a careful comparison.

**Comment by abegong** [comment]
I've manually checked the diffs for the three files with material changes---they're all identical to those in #2222. Closing this PR, since it looks like there's nothing original here.

**Pull request closed without merging**
